app/root.tsx

```diff
@@ -26,9 +24,12 @@ export function Layout({ children }: { children: React.ReactNode }) {
           skip to main content
         </a>
         <Navbar />
-        <main className="mx-auto relative mt-20 bg-white" id="main">
-          {children}
-        </main>
+        <Sidebar />
+        {!isHomepage && (
```

Comment:
This does not render the index route. You could give it a conditional style with a left margin the same width as your sidebar. something like:

~~~
<main className={`${isHomepage ? "ml-96" : "mx-auto"} relative mt-20 bg-white`} id="main">
  {children}
</main>
~~~